functions/assignOfflineContact.ts

```diff
@@ -62,6 +80,9 @@ const assignToAvailableWorker = async (
   if (completed.reservationStatus !== 'completed')
     return cleanUpTask(newTask, 'Error: reservation for task not completed.');
 
+  // eslint-disable-next-line no-console
+  if (retry) console.warn(`Needed ${retry} retries to get reservation`);
```

Comment:
Have you tested if this does produce output in the debugger or elsewhere?

Reply:
Yes! It is displayed in the Twilio debugger error log.

![Screenshot from 2021-04-15 12-55-12](https://user-images.githubusercontent.com/15805319/114900055-e03d8100-9de9-11eb-843a-bcaebfe76d15.png)